src/test/kotlin/org/rust/ide/console/RsConsoleFoldingTestBase.kt

```diff
@@ -0,0 +1,112 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.console
+
+import com.intellij.execution.ConsoleFolding
+import com.intellij.execution.impl.ConsoleViewImpl
+import com.intellij.execution.process.NopProcessHandler
+import com.intellij.execution.process.ProcessHandler
+import com.intellij.execution.ui.ConsoleViewContentType
+import com.intellij.openapi.extensions.ExtensionPoint
+import com.intellij.openapi.util.Disposer
+import org.rust.RsTestBase
+import org.rust.fileTreeFromText
+
+open class RsConsoleFoldingTestBase : RsTestBase() {
+    private var myConsole: ConsoleViewImpl? = null
+
+    public override fun setUp() {
+        super.setUp()
+
+        val console = createConsole()
+        console.setUpdateFoldingsEnabled(true)
+        myConsole = console
+
+        val point: ExtensionPoint<ConsoleFolding> = ConsoleFolding.EP_NAME.point
```

Comment:
The `point` is available since 2020.2.

Reply:
Is there an alternative in 2020.1, or should I disable the tests for 2020.1?

Reply:
I couldn't get the folding in tests to work, so I tried a lot of things :D In the end it turned out to be caused by missing `setUpdateFoldingsEnabled`, but then I forgot to remove the extension point.